src/wasm2asm.h

```diff
@@ -63,6 +63,13 @@ void flattenAppend(Ref ast, Ref extra) {
   }
 }
 
+enum NameScope {
```

Comment:
how about `enum class NameScope { Global, Local, Label, Max }` (then accessed by `NameScope::Global` etc)?

Reply:
Ah I originally tried this and it clashed with `Global`, but I've done the rename and otherwise renamed usage of `Global` to `class Global`